documentation/projects/index.md

```diff
@@ -0,0 +1,30 @@
+# Projects
+
+```{figure} /_static/project-tracker-screenshot.png
+---
+class: with-border
+---
+
+All Openverse projects can be viewed in the
+[Openverse Project Tracker](https://github.com/orgs/WordPress/projects/70?query=is%3Aopen+sort%3Aupdated-desc).
+```
+
+## Guides
+
+```{toctree}
+:titlesonly:
+:glob:
+:maxdepth: 2
+
+planning
+templates/*
+```
+
+## Proposals
+
+```{toctree}
+:glob:
+:maxdepth: 1
+
+proposals/**/index
```

Comment:
Nice! I was hoping it was possible to use a glob for this. By doing two stars, sub-folders will also get added at the same level if they have an `index.md`. Is that intentional? If this is changed to a single blob it doesn't do that, but it would make projects with sub-project directories responsible for adding a similar `*/index` glob. Not sure which is preferable but I kind of like being able to preserve project nesting on the toc side if we're doing so on the directory level.

```suggestion
proposals/*/index
```